proposals/2020/NMEC Implementation.md

```diff
@@ -0,0 +1,193 @@
+- Savings uncertainty: Is this what is defined by equations 4-6 and 4-7 in [1]? Is this the same across CalTrack methods / CPUC NMEC rulebook? #TODO
+
+Potentially?
+- Cutoff thresholds (for CVRMSE or NDBE)
+- 
+
+### Approach
+- Two `auc:Scenario`s will be used, one to represent the baseline period and one to represent the reporting period.  For situations where multiple reporting periods are considered (i.e. in the case where an energy simulation is used to produce synthetic data for different potential retrofit scenarios), additional reporting period scenarios can be defined.
+- The concept of an `auc:DerivedModel` is introduced.  It represents the generic concept of either a baseline or reporting model (referred to as an adjustment model in [3]).
+- The `auc:DerivedModel` complex type will be able to be instantiated as a child of the following elements:
+    - `auc:Scenario/auc:ScenarioType/auc:CurrentBuilding`
+        - It is expected that models for baseline periods would be embedded here (regardless if modeled or measured)
+        - `auc:DerivedModelPerformance` and child elements should be captured here
+        - All elements starting with `auc:BaselinePeriod` should be captured here
+    - `auc:Scenario/auc:ScenarioType/auc:PackageOfMeasures`
+        - It is expected that models for reporting periods would be embedded here (regardless if modeled or measured)
+        - All elements starting with `auc:ReportingPeriod` should be captured here
+- Multiple `auc:DerivdeModels` should be able to be captured, as users may want to test different `auc:ModelType`s
+- The `auc:DerivedModel` will be a complex type that would look something like:
+```xml
+  <auc:DerivedModels>
+    <auc:DerivedModel ID="Model1">
+      <auc:DerivedModelPeriod>[Baseline, Reporting]</auc:DerivedModelPeriod>
+      <auc:DerivedModelBaselineID/>
+      <auc:DerivedModelInputs>
+        <auc:ModelType>[2P SLR, 3P heating CPM, 3P cooling CPM, 4P CPM, 5P CPM]</auc:ModelType>
+        <auc:NormalizationMethod>[Forecast, Backcast, Standard Conditions]</auc:NormalizationMethod>
+        <auc:BaselinePeriodStartDate>2016-12-01</auc:BaselinePeriodStartDate>
+        <auc:BaselinePeriodEndDate>2017-12-01</auc:BaselinePeriodEndDate>
+        <auc:ReportingPeriodStartDate/>
+        <auc:ReportingPeriodEndDate/>
+        <auc:IntervalFrequency>Monthly</auc:IntervalFrequency>
+        <auc:ResponseVariable>
+          <auc:EnergyResource>Electricity</auc:EnergyResource>
+          <auc:EnergyResourceUnits>kWh</auc:EnergyResourceUnits>
+          <auc:EndUse>All end uses</auc:EndUse>
+        </auc:ResponseVariable>
+        <auc:ExplanatoryVariables>
+          <auc:ExplanatoryVariable>
+            <auc:ExplanatoryVariableName>[Temperature, Relative Humidity, GHI, DHI, DNI, Time-of-week, Time-of-day,...]</auc:ExplanatoryVariableName>
+            <auc:ExplanatoryVariableUnits>[Fahrenheit, Percent, W/m2, Hour of week, Hour of day, ...]</auc:ExplanatoryVariableUnits>
```

Comment:
is this going to be a constrained list?

Reply:
The enums are constrained, but you can have any number of ExplanatoryVariables

Reply:
See updated proposal.